modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -769,6 +804,90 @@ private DataDefinition _createDataDefinition(
 		return dataDefinition;
 	}
 
+	private void _testCopiedDataDefinitionDataLayout(
+		DataLayout dataLayout1, DataLayout dataLayout2) {
+
+		String[] dataLayoutColumnFieldNames1 = {};
+
+		for (DataLayoutPage dataLayoutPage : dataLayout1.getDataLayoutPages()) {
+			for (DataLayoutRow dataLayoutRow :
+					dataLayoutPage.getDataLayoutRows()) {
+
+				for (DataLayoutColumn dataLayoutColumn :
+						dataLayoutRow.getDataLayoutColumns()) {
+
+					dataLayoutColumnFieldNames1 =
+						dataLayoutColumn.getFieldNames();
+				}
+			}
+		}
+
+		String[] dataLayoutColumnFieldNames2 = {};
+
+		for (DataLayoutPage dataLayoutPage : dataLayout2.getDataLayoutPages()) {
+			for (DataLayoutRow dataLayoutRow :
+					dataLayoutPage.getDataLayoutRows()) {
+
+				for (DataLayoutColumn dataLayoutColumn :
+						dataLayoutRow.getDataLayoutColumns()) {
+
+					dataLayoutColumnFieldNames2 =
+						dataLayoutColumn.getFieldNames();
```

Comment:
@devpedrotavares This way `dataLayoutColumnFieldNames2` will only have one value, you should add to the array instead.

Reply:
Done!